speaker-generator.js

```diff
@@ -0,0 +1,69 @@
+  },
+
+  {
+    image: './images/people/maxime-profile.jpg',
+    name: 'Maxime Dubois',
+    position: 'Synthesizer expert and historian',
+    description: 'Maxime is a renowned expert on vintage synthesizers and the history of electronic music. He will give a talk on the evolution of the synthesizer and its impact on modern music.',
+  },
+
+  {
+    image: './images/people/zoe-profile.jpg',
+    name: 'Zoe Ramirez',
+    position: 'Live performance specialist',
+    description: 'Zoe is an expert in live electronic music performance, having performed at major festivals around the world. She will share her insights on creating dynamic, engaging performances using live instruments, software, and effects.',
+  },
+
+  {
+    image: './images/people/jake-profile.jpg',
+    name: 'Jake Patel',
+    position: 'Music software engineer',
+    description: 'Jake is a software engineer who has worked on some of the most popular music software and plugins in the industry. He will discuss the latest developments in music software and how they are changing the way we create and produce electronic music.',
+  },
+
+  {
+    image: './images/people/svetlana-profile.jpg',
+    name: 'Svetlana Petrova',
+    position: 'Experimental music composer',
+    description: 'Svetlana is an experimental music composer who has explored the boundaries of electronic music in her work. She will discuss her unique approach to composition and offer insights on pushing the limits of the genre.',
+  },
+
+  {
+    image: './images/people/alexei-profile.jpg',
+    name: 'Alexei Ivanov',
+    position: 'Music journalist and historian',
+    description: 'Alexei is a respected music journalist and historian who has covered electronic music for over two decades. He will share his extensive knowledge of the genre, its history, and its future, and offer insights on how to stay up-to-date with the latest trends and developments.',
+  },
+];
+
+function generateSpeakers() {
+  const sectionSpeaker = document.getElementById('featured-speakers');
+  const newSection = document.createElement('div');
+  newSection.id = 'speaker-profiles';
+  newSection.classList.add('pt-5', 'mt-4');
+
+  for (let i = 0; i < speakers.length; i += 1) {
+    const newProfile = document.createElement('div');
+    newProfile.id = 'person-profile';
+    newProfile.classList.add('pb-5');
+    if (i >= 2) {
+      newProfile.classList.add('hiding');
+      newProfile.style.display = 'none';
+    }
+    newProfile.innerHTML = `<img src=${speakers[i].image} class='person-image' alt='${speakers[i].name}'>
+                            <div class='person-text'><h3>${speakers[i].name}</h3>
+                            <h4>${speakers[i].position}</h4>
+                            <p>${speakers[i].description}</p></div>`;
+    newSection.appendChild(newProfile);
+  }
```

Comment:
- [ ] Kindly include the missing `grey line` shown in the image below 👇 so that our design looks similar to the given template. 



- [ ] Requirement screenshot

![image](https://user-images.githubusercontent.com/78251927/233806970-182f62be-085e-4669-82d7-b9a685962bc1.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/233806896-d9774a45-ecae-4b6c-b5d1-908e6f3fe69d.png)

Template
![image](https://user-images.githubusercontent.com/78251927/233806921-d4f914bb-caa5-4e22-b81c-6574aee0bcca.png)


# Mobile

Please kindly adjust your styling so the Layout of this section looks identical to the template

Your design
![image](https://user-images.githubusercontent.com/78251927/233807347-ff60a8e0-03e7-43fd-8fd8-b478be6d6293.png)

Template
![image](https://user-images.githubusercontent.com/78251927/233807403-7ddd7e19-3008-4292-964b-07d7f34312d5.png)

Part of the learning objective of the capstone project is to demonstrate our understanding of how to use medium-fidelity wireframes to create a UI.

Reply:
hello @abdulhamiid , thank you for your review. can you help me with this? I understand that I should follow the design but how should I fit all of the content without it being distorted? should I just make the entire element really tiny?